src/FileIndexer.ts

```diff
@@ -141,16 +141,17 @@ export class FileIndexer {
     sym: ts.Symbol,
     symbol: LsifSymbol
   ): void {
+    const documentation = [
```

Comment:
The diff in the snapshots was difficult to understand with this bugfix so I included it in the PR